src/operator/nn/dnnl/dnnl_act.cc

```diff
@@ -70,9 +70,9 @@ bool SupportDNNLLeakyRelu(const LeakyReLUParam& param, const NDArray& input) {
 }
 
 bool SupportQuantizedDNNLAct(const ActivationParam& param) {
-  // TODO(zhennan): Add more activation type when dnnl supports.
-  //                Remove this when it's identity to SupportDNNLAct.
-  return param.act_type == activation::kReLU;
+  // Although it is the same as SupportDNNLAct i left it here, so when new activations
```

Comment:
I understand what you mean but would still prefer this removed. I believe that a developer will create separate function for more sophisticated functions if needed.